src/devices/Mcp23xxx/Mcp23xxx.cs

```diff
@@ -205,54 +206,41 @@ public virtual void Dispose()
         {
             _masterGpioController?.Dispose();
             _masterGpioController = null;
-            _device?.Dispose();
+            _bus?.Dispose();
```

Comment:
I think this is missing `_bus = null` after

Reply:
I can't as I made it `readonly`. It begs the question of what is the right thing to do with disposable members. `readonly` has value and setting to null here doesn't _guarantee_ that it can't be reused in threaded scenarios.

Reply:
I'm going to poke around on this subject some more to see if we have / can establish guidelines.